### PR TITLE
Few fixies i fogot to push

### PR DESCRIPTION
**Pull request by abhiram555**
these are the few fixes you told me to do in comession channel. like change advancement namespaced key to normal one.

### CODE REVIEW

**Comment by abhiram555** [comment]
#6

**Comment by abhiram555** [comment]
and fixed player not spawn on playing world

**Comment by regulad** [comment]
> 
> 
> #6
image
this is a setting on github, you know :p

What is the placeholder for the translated advancement name?

**Comment by abhiram555** [comment]
> > #6
> > image
> > this is a setting on github, you know :p
> 
> What is the placeholder for the translated advancement name?

old one didn't change placeholders

**Review by regulad — COMMENTED** [review]
> 
> 
> > > #6
> > image
> > this is a setting on github, you know :p
> > 
> > 
> > What is the placeholder for the translated advancement name?
> 
> old one didn't change placeholders

there need to be 2 placeholders for `%id%` and `%name%`. See #6. Make sure you finish all parts of #6 before approving this PR. @abhiram555

File: src/main/java/de/teddy/advancementhunt/gamestates/IngameState.java [hunk 1]
```diff
@@ -34,7 +34,8 @@ public void start() {
 		}
 		this.advancement_id =  AdvancementHunt.getInstance().getAdvancement_id();
 
-		AdvancementHunt.getInstance().getMessageManager().sendMessageReplace(fleeingPlayer, MessageType.ISFLEEING_PLAYER,"%id%",this.advancement_id);
+		// Fixed Advancement using namspaced key ()
+		AdvancementHunt.getInstance().getMessageManager().sendMessageReplace(fleeingPlayer, MessageType.ISFLEEING_PLAYER,"%id%",this.advancement_id.split("/")[1]);
```
[the remark this answers is not in the document]
Comment (abhiram555) [reply src/main/java/de/teddy/advancementhunt/gamestates/IngameState.java:38]: oh like %id% %name%?

[the remark this answers is not in the document]
Comment (regulad) [reply src/main/java/de/teddy/advancementhunt/gamestates/IngameState.java:38]: See https://github.com/squirrelnetmc/AdvancementHunt/pull/11#pullrequestreview-560493137

**Pull request closed without merging**